pkgs/applications/audio/mopidy/podcast.nix

```
@@ -0,0 +1,26 @@
+{ lib, python3Packages, mopidy }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mopidy-podcast";
+  version = "3.0.0";
+
+  src = python3Packages.fetchPypi {
+    inherit version;
+    pname = "Mopidy-Podcast";
+    sha256 = "1z2b523yvdpcf8p7m7kczrvaw045lmxzhq4qj00dflxa2yw61qxr";
+  };
+
+  propagatedBuildInputs = [
+    mopidy
+    python3Packages.cachetools
+    python3Packages.uritools
+  ];
+
+  doCheck = false;
```

```suggestion
  checkInputs = with python3Packages; [
    pytestCheckHook
  ];
```

We really want tests enabled for python modules and application, to have some additional guarantees that they work.

---

Thanks much for your suggestions! I've updated the maintainer + used a real name account, and added the changes requested above.